<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsideComm</class>
 <widget class="QWidget" name="InsideComm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>715</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>-1</y>
     <width>800</width>
     <height>700</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>通讯设置</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>149</y>
       <width>471</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>通讯设置</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>56</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>网口模式:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_IP">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>133</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>服务器IP地址:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Port">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>173</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>服务器端口号:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Mode">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>54</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IP_1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>129</y>
        <width>41</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>192</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>204</x>
        <y>142</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IP_2">
      <property name="geometry">
       <rect>
        <x>212</x>
        <y>129</y>
        <width>41</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>168</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>141</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IP_3">
      <property name="geometry">
       <rect>
        <x>264</x>
        <y>129</y>
        <width>41</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>309</x>
        <y>143</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IP_4">
      <property name="geometry">
       <rect>
        <x>317</x>
        <y>129</y>
        <width>41</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Port_number">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>168</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>2000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_connect">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>96</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>设备地址:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_dev_Addr">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>91</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>数据设置</string>
    </attribute>
    <widget class="QPushButton" name="btnDown">
     <property name="geometry">
      <rect>
       <x>507</x>
       <y>37</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>下移</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>37</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>307</x>
       <y>93</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 153, 153);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetData">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>122</y>
       <width>781</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUp">
     <property name="geometry">
      <rect>
       <x>427</x>
       <y>37</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>上移</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>37</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>93</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 153, 153);</string>
     </property>
     <property name="text">
      <string>   名称  </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxModbusTool">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>42</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>677</x>
       <y>93</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 153, 153);</string>
     </property>
     <property name="text">
      <string>详细</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>41</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Modubs工具:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
